local-cli/cliEntry.js

```diff
@@ -137,7 +137,7 @@ function run() {
 
   commander.parse(process.argv);
 
-  const isValidCommand = commands.find(cmd => cmd.name === process.argv[2]);
+  const isValidCommand = commands.find(cmd => cmd.name.split(" ")[0] === process.argv[2]);
```

Comment:
Looks like this might be breaking all usages of `link ABCD` with the RC? If yes, that has to be cherry-picked into stable release.